modyn/config/schema/pipeline.py

```diff
@@ -417,6 +417,24 @@ class DataConfig(BaseModel):
         description="Function to tokenize the input. Must be a class in modyn.models.tokenizers.",
     )
 
+    @field_validator("bytes_parser_function", mode="before")
+    @classmethod
+    def validate_bytes_parser_function(cls, value: str) -> str:
+        try:
+            res = deserialize_function(value, "bytes_parser_function")
+            if not callable(res):
+                raise ValueError("Function 'bytes_parser_function' must be callable!")
+        except AttributeError as exc:
+            raise ValueError("Function 'bytes_parser_function' could not be parsed!") from exc
+        return value
+
+    @property
+    def bytes_parser_function_deserialized(self) -> Callable:
```

Comment:
If we introduce this property, should we use it in the dataset implementations of the `OnlineDataset` and the `EvaluationDataset`(please don't freak out - they are messy with lots of code duplication haha, see #275)?

Reply:
yes, then the caller does not need to deserialize it themselves. I will leave this decision of whether, when, and whether to update you. I don't intend to touch that ;)